### ISSUE TITLE
no-overwrite IT fails

### ISSUE THREAD

**Issue by lasselindqvist** [mojohaus/flatten-maven-plugin#136]
no-overwrite IT fails. This blocks the release until fixed.

```
    [INFO] Building: no-overwrite\pom.xml
    [INFO] run script verify.groovy
    [INFO] ..FAILED (6.2 s)
    [INFO]   The post-build script did not succeed. assert System.currentTimeMil
lis() - flattendPom.lastModified() > 30*1000
                  |                   | |           |              |   |
                  1585152197487       94|           1585152197393  |   30000
                                        |                          false
flatten-maven-plugin\target\it\no-overwrite\.flattened-pom.xml
```

**Comment by olamy** [comment]
and doesn't fail in travis :(

**Comment by olamy** [comment]
fixed the issue. You may using maven 3.6 and travis was using 3.5.4
some change in writing pom :) 
now
`<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0">`

before 
`<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0">`

1 byte difference :)

**Comment by stephaniewang526** [comment]
Seems like this fix is causing issues in Maven version before 3.6.3.

We tested in 3.6. and 3.5.2 and they both failed because of "https".

This seems to only work for 3.6.3.

We could upgrade our Maven version but should this be fixed for versions lower than 3.6.3?

@nuoyuyang @lasselindqvist

**Comment by olamy** [comment]
well If you have bandwidth to add a test if <3.6.3 then `http` let's do it
But I would consider this as very very very minor as it's only for testing purpose and maven 3.6.3 is right using https rather than http.
